packages/server-wallet/src/wallet/store.ts

```diff
@@ -455,101 +444,79 @@ export class Store {
     return LedgerRequest.getPendingRequests(ledgerChannelId, tx || this.knex);
   }
 
+  /**
+   * Add a new state into the database.
+   *
+   * @param channelId - the channel ID the state refers to
+   * @param state - the new signed state to be added
+   * @param tx - an existing database transaction context
+   *
+   * This function serves as both a data validator (ensure the state is valid) and
+   * an efficient operator on the database.
+   */
   async addSignedState(
     channelId: string,
-    maybeChannel: Channel | undefined,
```

Comment:
By ripping out this parameter, you are making this function go to the DB _for sure_. Previously, I could skip that if  I already had the channel?

Reply:
It was never used in any instance of the codebase, so I removed it for that reason